tools/topology/sof/pipe-processing-playback.m4

```diff
@@ -57,6 +57,15 @@ CONTROLBYTES_PRIV(PP_RUNTIME_PARAMS,
 `	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00"'
 )
 
+# Post process Bytes control for runtime config
```

Comment:
Can you comment here what this does, what structure it maps too and whether any tool was used in it's creation.